src/Entity/EntityId.php

```diff
@@ -18,6 +18,13 @@
 	protected $serialization;
 
 	/**
+	 * @param $serialization
```

Comment:
this is always a string, right? then you could have `@param string $serialization` here